ansible_base/authentication/common.py

```diff
@@ -276,6 +276,9 @@ def get_or_create_authenticator_user(user_id, user_details, authenticator, extra
 
 
 def update_user_claims(user, database_authenticator, groups):
+    if not user:
+        return
```

Comment:
Just to be explicit can we `return None` here.

Reply:
Sure.